instagram-app/src/components/PostContainer/Post.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PostHead from './PostHead';
+import CommentSection from '../CommentSection/CommentSection';
+import './Post.css';
+
+const Post = props => {
+    console.log("Post", props)
+    return(
+        <div>
+            <PostHead 
+                username={props.postCont.username}
+                thumbnailImg={props.postCont.thumbnailUrl} />
```

Comment:
In the PostHead component, it appears you expect props to be passed as *thumbnailUrl* instead of *thumbnailImg*

Reply:
Fixed :)